batch/batch-prompt.md

```
@@ -306,13 +306,14 @@ next_action: "{one concrete next step}"
-12. Escribe HTML a `/tmp/cv-candidate-{company-slug}.html`
+12. Escribe HTML a `output/cv-candidate-{company-slug}.html` (NO en /tmp — el HTML registrado es la fuente de regeneración del dashboard)
 13. Ejecuta:
 ```bash
 node generate-pdf.mjs \
-  /tmp/cv-candidate-{company-slug}.html \
+  output/cv-candidate-{company-slug}.html \
   output/cv-candidate-{company-slug}-{{DATE}}.pdf \
-  --format={letter|a4}
+  --format={letter|a4} \
+  --report={num}
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)